index.html

```diff
@@ -515,6 +515,19 @@ <h1>
 These are elaborated in <a href="#did-documents"></a>.
     </p>
 
+    <section>
+      <h2>
+Identifier
+      </h2>
+      <p>
+Identifiers are used in the data model to identify specific instances of people,
+keys, organizations, services, entities, and things in general. Identifiers are
+typically URLs, but can also be URNs. Non-URN based identifiers are not advised
```

Comment:
I didn't follow this... URLs (non-URNs) are advised but this suggests (probably erroneously) otherwise.

Perhaps it should say this instead:

```
While the data model allows for other URI-based identifiers, they are not advised as they tend to not be easily compatible with Internet-based technologies.
```

But I'm not even sure if that's what was intended.

Reply:
I think it's a simple typo. "Non-URN based" probably ought to have been "Non-URI based" (or "Non-IRI based"). That said, your rephrase seems clearer to me.

Reply:
Yep, typo, fixing.

Reply:
Fixed in https://github.com/w3c/did-spec/pull/29/commits/a22f0c8e78d4158ba47ba3e1345c8664b04aede2